app/assets/stylesheets/components/_language_selector.scss

```diff
@@ -0,0 +1,16 @@
+.language-selector__select {
+  margin-bottom: $baseline-unit*2;
+}
+
+.language-selector__delete {
+  margin-left: 0.5em;
+  cursor: pointer;
```

Comment:
Maybe a `button`, even if they look like links? Although it doesn't take you somewhere, it does "do" something?